cypress/integration/multiple-choice.test.js

```diff
@@ -1,7 +1,10 @@
 import { phonePost, phoneListen, getAndClearLastPhoneMessage } from "../support";
 
 context("Test multiple-choice interactive", () => {
+  let i = 0;
+
   beforeEach(() => {
+    if (i++ > 0) cy.wait(3000);
```

Comment:
These wait times are pretty long. Is that the smallest value that helps?

Reply:
As I mentioned elsewhere:

>I did not conduct an extensive experiment, but my first tests were with one and two seconds, which failed, with three being the first value that allowed for a successful test run. Three worked fairly consistently but would occasionally fail on certain tests, in which case I bumped it up to four. It's possible that with additional effort they could be fine-tuned to test content, but I'm not sure it'd be worth it.